kuma/wiki/tasks.py

```diff
@@ -37,6 +37,12 @@ def render_document(pk, cache_control, base_url):
     return document.rendered_errors
 
 
+@task()
+def clear_render_stale_document_lock():
+    lock = MemcacheLock('render-stale-documents-lock', expires=60 * 60)
+    lock.release()
+
+
 @task(throws=(StaleDocumentsRenderingInProgress,))
 def render_stale_documents(log=None):
     """Simple task wrapper for rendering stale documents"""
```

Comment:
I've renamed it to `locked` and made it a method. So `lock.locked` returns either `True` or `False`